source/computer/fedora-setup.rst

```diff
@@ -11,13 +11,6 @@ Fedora 配置指南
    本节内容适用于 **Fedora 33 Workstation**\，不一定适用于其他 Fedora 版本。
    建议用户总是选择 Fedora 最新版本。如遇到问题，欢迎反馈。
 
-.. note::
-
-   本节大部分软件都通过命令行安装。在桌面或菜单栏中找到并点击 “Terminal” 图标以启动终端，
-   然后在终端中输入命令并按下 :kbd:`Enter` 键即可执行该命令。
-
-----
-
```

Comment:
The note makes no sense until the system is installed, so I move the note after installing this  OS.